src/applications/dhp-connected-devices/containers/App.jsx

```diff
@@ -21,26 +24,58 @@ export default function App() {
             your VA care team. If you have concerns about any specific shared
             data, you must contact your care team directly.
           </p>
-          <div className="schemaform-title">
-            <h2>Connected devices</h2>
-          </div>
-          <va-alert
-            close-btn-aria-label="Close notification"
-            status="continue"
-            visible
-          >
-            <h3 slot="headline">Please sign in to connect a device</h3>
-            <div>
-              Sign in with your existing ID.me, DS Logon, or My HealtheVet
-              account. If you don't have any of these accounts, you can create a
-              free ID.me account now.
-            </div>
-            <div>
-              <button type="button" className="usa-button">
-                Sign in or create an account
-              </button>
-            </div>
-          </va-alert>
+          {/* Show sign in button if user not logged in */}
+          {!isLoggedIn && (
+            <>
+              <h2>Connected devices</h2>
+              <va-alert
+                close-btn-aria-label="Close notification"
+                status="continue"
+                visible
+              >
+                <h3 slot="headline">Please sign in to connect a device</h3>
+                <div>
+                  Sign in with your existing ID.me, DS Logon, or My HealtheVet
+                  account. If you don't have any of these accounts, you can
+                  create a free ID.me account now.
+                </div>
+                <div className="button">
+                  <UserNav
+                    isHeaderV2
+                    customText="Sign in or create an account"
+                  />
+                </div>
+              </va-alert>
+            </>
```

Comment:
Does it make sense to extract this part into a component?

Reply:
@yonidoronpeters it might, but if you notice on line 43 we're using a component called `<UserNav />`. Are you suggesting we might be able to pull all of this code into a component, including the component on line 43?

Reply:
You can nest them I think! Good call Yoni